docs/site/DEVELOPING.md

```diff
@@ -80,6 +81,25 @@ Please note that we are automatically running the build from `pretest` script,
 therefore you should not need to run this command as part of your
 [red-green-refactor cycle](http://www.jamesshore.com/Blog/Red-Green-Refactor.html).
 
+## Utilizing the monorepo
+
+The monorepo's source code can be utilized for testing new features or bug fixes
+withing LoopBack itself or applications are dependent on LoopBack.
```

Comment:
Suggestion:
Source code from the monorepo can be utilized as LoopBack 4 packages that are not published to npm registry yet. This is useful to test or debug your applications or extensions with project dependencies on LoopBack 4 against a local git repo and branch of `loopback-next`.